src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustCorporateValidatorTest.java

```diff
@@ -182,12 +184,45 @@ void testErrorReportedWhenDateBecameInterestedPersonIsInTheFuture() {
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
+    @Test
+    void testNoErrorReportedWhenDateBecameInterestedPersonIsAfterTrustCreationDate() {
+        trustDataDtoList.get(0).getCorporates().get(0).setType(BeneficialOwnerType.INTERESTED_PERSON.getValue());
+        trustDataDto.setCreationDate(LocalDate.of(2000, 1,1));
+        trustDataDtoList.get(0).getCorporates().get(0).setDateBecameInterestedPerson(LocalDate.of(2000,1,2));
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        assertFalse(errors.hasErrors());
+    }
+
+
+    @Test
+    void testNoErrorReportedWhenDateBecameInterestedPersonIsOnTrustCreationDate() {
+        trustDataDtoList.get(0).getCorporates().get(0).setType(BeneficialOwnerType.INTERESTED_PERSON.getValue());
+        trustDataDto.setCreationDate(LocalDate.of(2000, 1,1));
+        trustDataDtoList.get(0).getCorporates().get(0).setDateBecameInterestedPerson(LocalDate.of(2000,1,2));
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        assertFalse(errors.hasErrors());
+    }
+
+    @Test
+    void testErrorReportedWhenDateBecameInterestedPersonIsBeforeTrustCreationDate() {
+        trustDataDtoList.get(0).getCorporates().get(0).setType(BeneficialOwnerType.INTERESTED_PERSON.getValue());
+        trustDataDto.setCreationDate(LocalDate.of(2000, 1,2));
+        trustDataDtoList.get(0).getCorporates().get(0).setDateBecameInterestedPerson(LocalDate.of(2000,1,1));
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+
+        String qualifiedFieldName = getQualifiedFieldName(PARENT_FIELD,
+                TrustCorporateDto.DATE_BECAME_INTERESTED_PERSON_FIELD);
```

Comment:
Assuming tests do need amending, the qualified field name here would be the 'ceased date'.

Reply:
Remved